cmd2/AutoCompleter.py

```diff
@@ -0,0 +1,796 @@
+# coding=utf-8
+import argparse
+import re as _re
+import sys
+from argparse import OPTIONAL, ZERO_OR_MORE, ONE_OR_MORE, REMAINDER, PARSER, ArgumentError, _
+from .rl_utils import rl_force_redisplay
+try:
+    from typing import List, Dict, Tuple, Callable, Union
+except:
```

Comment:
I'm going to just take this out since it didn't actually fix the problem. The addition of a dependency on the typing backport to Python 3.4 fixes this.